frontend/server/config.default.php

```diff
@@ -91,6 +91,14 @@
 try_define('OMEGAUP_ENABLE_REJUDGE_ON_PROBLEM_UPDATE', true);
 try_define('OMEGAUP_GRADER_FAKE', false);
 
+# ####################################
+# COURSE CLONE CONFIG
+# ####################################
+try_define(
+    'OMEGAUP_COURSE_CLONE_SECRET_KEY',
+    '6f8xSU_xkrelmCTSahbbxl3PRovgAfkrThyrqQ9JesE='
```

Comment:
```suggestion
    '6f8xSU_xkrelmCTSahbbxl3PRovgAfkrThyrqQ9JesE'
```

ya encontré el problema: Paseto usa una [versión modificada de Base64](https://en.wikipedia.org/wiki/Base64#Variants_summary_table) (usa [RFC 4648 §5](https://tools.ietf.org/html/rfc4648#section-5)) que no es compatible con [`base64_decode()`](http://php.net/base64_decode) (usa [RFC 2045](https://tools.ietf.org/html/rfc2045)): está ignorando el `_` de la llave, causando un caracter menos.

Si nos limitamos a usar las APIs de Paseto para encodificar y decodificar la llave, todo funciona como debería.

Reply:
Ok. Se regresa como estaba.